From 2963aa7a441384c61c056670ca2cd44ee0fcd4a1 Mon Sep 17 00:00:00 2001
From: perillamint <perillamint@gentoo.moe>
Date: Wed, 23 Sep 2015 22:01:06 +0900
Subject: [PATCH 3/5] sudo without password might be bad idea.

---
 scripts/mkmodimg.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/scripts/mkmodimg.sh b/scripts/mkmodimg.sh
index e4cc9f1..2d87212 100755
--- a/scripts/mkmodimg.sh
+++ b/scripts/mkmodimg.sh
@@ -28,7 +28,7 @@ cd ${TMP_DIR}/lib
 mkdir -p tmp
 dd if=/dev/zero of=${BIN_NAME} count=${BIN_SIZE} bs=1024
 mkfs.ext4 -q -F -t ext4 -b 1024 ${BIN_NAME}
-sudo -n mount -t ext4 ${BIN_NAME} ./tmp -o loop
+sudo mount -t ext4 ${BIN_NAME} ./tmp -o loop
 if [ "$?" != "0" ]; then
 	echo "Failed to mount (or sudo)"
 	exit 1
-- 
2.4.9

